src/components/Admin/Users.js

```diff
@@ -0,0 +1,244 @@
+            });
+        }
+        if(stat===-1) {
+            notification.open({
+                message: 'Delete Failed',
+                description:
+                  'User not deleted',
+                icon: <Icon type="closed" style={{ color: '#108ee9' }} />,
+            });
+        }
+      };
+
+      deleteAllServicesNotification = (stat) => {
+        if(stat===1) {
+            notification.open({
+                message: 'Delete Successful',
+                description:
+                  'All Services of vendor deleted',
+                icon: <Icon type="check-circle" style={{ color: '#108ee9' }} />,
+            });
+        }
+      };
+
+      getColumnSearchProps = dataIndex => ({
+        filterDropdown: ({ setSelectedKeys, selectedKeys, confirm, clearFilters }) => (
+          <div style={{ padding: 8 }}>
+            <Input
+              ref={node => {
+                this.searchInput = node;
+              }}
+              placeholder={`Search ${dataIndex}`}
+              value={selectedKeys[0]}
+              onChange={e => setSelectedKeys(e.target.value ? [e.target.value] : [])}
+              onPressEnter={() => this.handleSearch(selectedKeys, confirm)}
+              style={{ width: 188, marginBottom: 8, display: 'block' }}
+            />
+            <Button
+              type="primary"
+              onClick={() => this.handleSearch(selectedKeys, confirm)}
+              icon="search"
+              size="small"
+              style={{ width: 90, marginRight: 8 }}
+            >
+              Search
+            </Button>
+            <Button onClick={() => this.handleReset(clearFilters)} size="small" style={{ width: 90 }}>
+              Reset
+            </Button>
+          </div>
+        ),
+        filterIcon: filtered => (
+          <Icon type="search" style={{ color: filtered ? '#1890ff' : undefined }} />
+        ),
+        onFilter: (value, record) =>
+          record[dataIndex]
+            .toString()
+            .toLowerCase()
+            .includes(value.toLowerCase()),
+        onFilterDropdownVisibleChange: visible => {
+          if (visible) {
+            setTimeout(() => this.searchInput.select());
+          }
+        },
+        render: text => (
+          <Highlighter
+            highlightStyle={{ backgroundColor: '#ffc069', padding: 0 }}
+            searchWords={[this.state.searchText]}
+            autoEscape
+            textToHighlight={text.toString()}
+          />
+        ),
+      });
+    
+      handleSearch = (selectedKeys, confirm) => {
+        confirm();
+        this.setState({ searchText: selectedKeys[0] });
+      };
+    
+      handleReset = clearFilters => {
+        clearFilters();
+        this.setState({ searchText: '' });
+      };
+
+    render() {
+        const { msg } = this.state;
+        const { isAuthenticated, user, status, statusType } = this.props;
+
+        if(statusType==="deleteUser" && status===200) {
+            const stat=1;
```

Comment:
Do you need this? Why can't you directly pass the value in the function?

Reply:
@harsh-jindal  I've removed the extra variable;
if(statusType==="deleteUser" && status===200) {
    this.deleteUserNotification(1);
}
if(statusType==="deleteUser" && status==null) {
    this.deleteUserNotification(-1);
}